apps/annotation/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+SET(deps opencv_core opencv_highgui opencv_imgproc)
+ocv_check_dependencies(${deps})
+
+if(NOT OCV_DEPENDENCIES_FOUND)
+   return()
+endif()
+
+project(annotation)
+
+ocv_include_directories("${CMAKE_CURRENT_SOURCE_DIR}" "${OpenCV_SOURCE_DIR}/include/opencv")
+ocv_include_modules(${deps})
+
```

Comment:
Please, add here the line:
```{.cmake}
set(the_target opencv_annotation)
```
Otherwise, install commands will not work.
Also it will be better to replace `opencv_annotation` with `${the_target}` on lines 13,14 and 16.
Sorry, I've forgot to fix this before commit.